src/FavoriteOutputDefinition/Listing.php

```diff
@@ -15,16 +15,17 @@
 
 namespace Web2PrintToolsBundle\FavoriteOutputDefinition;
 
+/**
+ * @method self|null load()
+ * @method int getTotalCount()
+ */
 class Listing extends \Pimcore\Model\Listing\AbstractListing
 {
     /**
      * @var array
      */
     public $outputDefinitions;
 
-    /**
-     * @var array
-     */
     public function isValidOrderKey($key)
```

Comment:
removal and everything looks fine to me now, just missing this php docblock 
```suggestion
    /**
     * @param string $key
     *
     * @return bool
     */
    public function isValidOrderKey($key)
```